core/src/main/java/com/scalar/db/exception/transaction/TransactionNotFoundException.java

```diff
@@ -1,6 +1,10 @@
 package com.scalar.db.exception.transaction;
 
-public class TransactionNotFoundException extends TransactionException {
+/**
+ * An exception thrown when a transaction you are trying to resume is not found. You can retry the
+ * same transaction from the beginning in this case.
+ */
+public class TransactionNotFoundException extends RetriableTransactionException {
```

Comment:
Made `TransactionNotFoundException` extend `RetriableTransactionException` instead of `TransactionException`.